recipe/meta.yaml

```diff
@@ -1,99 +1,238 @@
+        - {{ pin_compatible('boost-cpp') }}
+        - poppler-data
+
+    test:
+      commands:
+        - pdfinfo -listenc
+        - pdfunite --help
+        - pdftocairo --help
+
+  - name: poppler-qt
+    build:
+      skip: true  # [ppc64le or s390x]
+      missing_dso_whitelist:
+        - $RPATH/libm.so.6
+        - $RPATH/libc.so.6
+        - $RPATH/ld-linux-x86-64.so.2
+        - $RPATH/libpthread.so.0
+        - $RPATH/poppler.dll
+        - $RPATH/ld64.so.1
+    script: install.sh  # [unix]
+    script: install.bat  # [not unix]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # Need these CDTs for Qt on Linux. Please keep them alphabetized!
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxscrnsaver-devel') }}   # [linux and not ppc64le]
+        - {{ cdt('libxtst-devel') }}         # [linux and not ppc64le]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux and not s390x]
+        - {{ cdt('mesa-libegl-devel') }}     # [linux and not ppc64le]
+        - {{ cdt('mesa-libgbm') }}           # [linux and not ppc64le]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libx11-devel') }}          # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('libxrender-devel') }}      # [linux]
+        - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+        - m2-msys2-runtime  # [win]
+        - {{ native }}pkg-config
+        - cmake
+        - {{ posix }}make
+        - ninja
+        - {{ posix }}patch
+        - perl 5.*
+        - gobject-introspection 1.*  # [not win]
+
+      host:
+        - python 3.9
+        - boost-cpp {{ boost_cpp }}
+        - cairo
+        - curl
+        - fontconfig
+        - freetype
+        - gettext  # [osx]
+        - glib
+        - jpeg
+        - lcms2
+        - libcurl
+        - libiconv
+        - libpng
+        - libtiff
+        - nss  # [not (win or s390x)]
+        - openjpeg
+        - qt-main  # [not (ppc64le or s390x)]
+        - zlib
+        - {{ pin_subpackage('poppler', exact=True) }}
+        - nspr  # [not win]
+      run:
+        - {{ pin_subpackage('poppler', exact=True) }}
+
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/pkgconfig/poppler-qt5.pc  # [not win]
+        - if not exist %LIBRARY_BIN%\\poppler.dll exit 1  # [win]
 
 about:
   home: https://poppler.freedesktop.org/
-  license: GPLv2
+  license: GPL-2.0-only
   license_family: GPL
   license_file: COPYING
-  summary: 'The Poppler PDF manipulation library.'
-
+  summary: The Poppler PDF manipulation library.
+  description: |
+    Poppler is a PDF rendering library based on the xpdf-3.0 code base.
+  doc_url: https://poppler.freedesktop.org/api/
```

Comment:
`doc_url` isn't correct. Should be one of:
cpp: https://poppler.freedesktop.org/api/cpp/
glib: https://poppler.freedesktop.org/api/glib/
qt5: https://poppler.freedesktop.org/api/qt5/
qt6: https://poppler.freedesktop.org/api/qt6/

or simply https://poppler.freedesktop.org/